simulation_parameters/microbe_stage/compounds.json

```diff
@@ -77,17 +77,33 @@
         "name": "OXYTOXY_NT",
         "volume": 1,
         "isCloud": false,
-        "isAlwaysUseful": true,
+        "isAgent": true,
         "isEnvironmental": false,
         "canBeDistributed": true,
-        "digestible": true,
+        "digestible": false,
```

Comment:
Does this change impact anything? For example engulf damage should not be broken.